esp/templates/program/modules/nametagmodule/selectoptions.html

```diff
@@ -47,36 +47,50 @@
 <strong>ID Type</strong>
 <ul>
   <li><input type="radio" name="type" value="students" id="students"
-       onclick="document.getElementById('volunteers').disabled = true;" checked="checked" />
+       onclick="document.getElementById('misc_info').disabled = true; document.getElementById('group').disabled = true;" checked="checked" />
       <label for="students">Students</label>
   </li>
   <li><input type="radio" name="type" value="teacher" id="teachers"
-       onclick="document.getElementById('volunteers').disabled = true;" />
+       onclick="document.getElementById('misc_info').disabled = true; document.getElementById('group').disabled = true;" />
       <label for="teachers">Teachers</label>
   </li>
+  <li><input type="radio" name="type" value="other" id="other"
+       onclick="document.getElementById('misc_info').disabled = true; document.getElementById('group').disabled = false;" />
+      <label for="teachers">Other Group</label>
+  </li>
+<select name="group" id="group" disabled>
```

Comment:
This should probably just go inside the `<li>`, perhaps to the right of "Other Group".